expan/core/statistics.py

```diff
@@ -89,8 +91,9 @@ def delta(x, y, assume_normal=True, percentiles=[2.5, 97.5],
         mu = _delta_mean(_x, _y)
         # Computing the confidence intervals
         if assume_normal:
-            c_i = normal_sample_difference(x=_x, y=_y, percentiles=percentiles,
-                                           relative=relative)
+            percentiles = [float(p)/num_tests if p < 50.0
+                           else 100-(100-float(p))/num_tests if p > 50.0 else p for p in percentiles]
+            c_i = normal_sample_difference(x=_x, y=_y, percentiles=percentiles, relative=relative)
         else:
             c_i, _ = bootstrap(x=_x, y=_y, percentiles=percentiles, nruns=nruns,
```

Comment:
and what about the non-normal distributions (I don't know either)?

Reply:
good question, but I am not sure about the correction for the bootstrap. Will investigate that.

Reply:
so, basically the question here is not about to correct for non-normal distribution or not (correction as it is does not depend on the distribution of the data), but whether to correct for the bootstrap or not. In our scenario it make sense to make the same correction of CIs as for the hypothesis testing, since here we report corrected confidence intervals and, basically, they should be approximately similar to CIs reported by hypothesis testing. Bootstrap is not a hypothesis testing though, but in our case it may be considered as another way of representing the data. Here we correct more in terms of the decision on how to select a confidence interval: the more kpis we observe the higher the chance that one kpi will give different results than others.  But if you have other ideas, I would be happy to discuss.